lms/templates/lti_launches/new_lti_launch.html.jinja2

```diff
@@ -1 +1,7 @@
-<iframe width="100%" height="100%" src="{{hypothesis_url}}" />
\ No newline at end of file
+<script type="application/json" class="js-hypothesis-config">{{ context.hypothesis_config|safe }}</script>
```

Comment:
The latest version of Jinja 2 has a [tojson](http://jinja.pocoo.org/docs/2.10/templates/#tojson) filter which I would advise using when embedding JSON strings in HTML. See the `to_json` filter in h for an explanation of why.

Reply:
Done!